tests/library/web-socket.spec.ts

```diff
@@ -175,9 +175,9 @@ it('should reject waitForEvent on socket close', async ({ page, server }) => {
       await ws.waitForEvent('framereceived');
       return ws;
     }),
-    page.evaluate(port => {
-      (window as any).ws = new WebSocket('ws://localhost:' + port + '/ws');
-    }, server.PORT)
+    page.evaluate(host => {
+      (window as any).ws = new WebSocket('ws://' + host + '/ws');
+    }, server.HOST)
```

Comment:
Can we remove `server.PORT` property entirely? That will force new tests to work with non-localhost out of the box.

Reply:
Still used in a lot of proxy/har/fetch tests - not sure if its worth it.